docs/tutorials/eda/eda-auto-analyze-interaction.ipynb

```diff
@@ -145,6 +145,90 @@
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "62e0cc5e",
+   "metadata": {},
+   "source": [
+    "Let's take a look at the two-way partial dependence plots to visualize any potential interactions between the two features. Here are some cases when it's a good idea to two-way PDP:\n",
+    "\n",
+    "* **Suspected interactions**: Even if two features are not highly correlated, they may still interact in the context of the model. If you suspect that there might be interactions between any two features, two-way PDP can help to verify the hypotheses.\n",
+    "\n",
+    "* **Moderate to high correlation**: If two features have a moderate to high correlation, a two-way PDP can show how the combined effect of these features influences the model's predictions. In this case, the plot can help reveal whether the relationship between the features is additive, multiplicative, or more complex.\n",
+    "\n",
+    "* **Complementary features**: If two features provide complementary information, a two-way PDP can help illustrate how the joint effect of these features impacts the model's predictions. For example, if one feature measures the length of an object and another measures its width, a two-way PDP could show how the area affects the predicted outcome.\n",
+    "\n",
+    "* **Domain knowledge**: If domain knowledge suggests that the relationship between two features might be important for the model's output, a two-way PDP can help to explore and validate these hypotheses.\n",
+    "\n",
+    "* **Feature importance**: If feature importance analysis ranks both features high in the leaderboard, it might be beneficial to examine their joint effect on the model's predictions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "08827afd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "auto.partial_dependence_plots(df_train, label='Survived', features=['Fare', 'Age'], two_way=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f7e9b205",
+   "metadata": {},
+   "source": [
+    "We can see these two features interact in the bottom left quadrant of the chart, but have almost no effect on each other in other areas. Areas where `Age < 45` and `Fare < 60` can be explored further."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "db4212bc",
+   "metadata": {},
+   "source": [
+    "Let's take a look interactions between the features with the highest importance. To do this, we'll fit a quick model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "745ed086",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "state = auto.quick_fit(train_data=df_train, label='Survived', render_analysis=False, return_state=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8628fe2f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "state.model_evaluation.importance"
```

Comment:
Just curious, what's the output of this line?

Reply:
feature importance of the trained model (previous cell trains one). [Website](http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-3079/d71b34a/tutorials/eda/eda-auto-analyze-interaction.html)

<img width="841" alt="image" src="https://user-images.githubusercontent.com/10080307/228317967-936a42e1-11dc-4e5c-b79f-04a2f94c547e.png">